stage1/init/pod.go

```diff
@@ -115,9 +115,7 @@ func quoteExec(exec []string) string {
 	}
 
 	var qexec []string
-	qexec = append(qexec, exec[0])
-	// FIXME(vc): systemd gets angry if qexec[0] is quoted
-	// https://bugs.freedesktop.org/show_bug.cgi?id=86171
+	qexec = append(qexec, `"`+exec[0]+`"`)
 
 	if len(exec) > 1 {
 		for _, arg := range exec[1:] {
```

Comment:
I thought the idea was we could use the same logic here for all args now. maybe I'm wrong. /cc @vcaputo